src/pubsub/message/sign.js

```diff
@@ -31,14 +31,16 @@ async function signMessage (peerId, message) {
 
 /**
  * Verifies the signature of the given message
+ *
  * @param {InMessage} message
- * @returns {Promise<Boolean>}
+ * @returns {Promise<boolean>}
  */
 async function verifySignature (message) {
   // Get message sans the signature
   const baseMessage = { ...message }
   delete baseMessage.signature
   delete baseMessage.key
+  // @ts-ignore - from is optional
```

Comment:
its safer to either make it required or protect with and if clause

Reply:
Per the [spec](https://github.com/libp2p/specs/tree/master/pubsub#the-message), it must be optional. So I will be adding an if clause